ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
I think we'll remove this model at some point (now?), so this import will break.
Not sure exactly what the proper way to deal with that situation in Django, but I guess we should load the model from the registry, more or less like in https://docs.djangoproject.com/en/1.10/ref/migration-operations/#runpython ?
Also, there are very few servers with the Setting model being used, and it's only for the server name, so it may be just a good idea to just delete the model and put back the server name by hand when we update.